backends/bmv2/simpleSwitch.cpp

```diff
@@ -36,6 +36,7 @@ namespace P4V1 {
 // need a class to generate simpleSwitch
 
 cstring jsonMetadataParameterName = "standard_metadata";
+cstring externFunctionPrefix = "__extern_";
```

Comment:
I'm not sure why this prefix is needed at all. I believe we did remove the automatic prepending of this prefix when registering the function in your bmv2 PR. Did you actually experience a name conflict when using this?

Reply:
I added this prefix from the get-go. But I will remove it. On second thought, a collision like this would probably appear in the p4 level as well, not just after compilation.